src/vos/tests/vts_dtx.c

```diff
@@ -388,10 +388,10 @@ vts_dtx_commit_visibility(struct io_test_args *args, bool ext, bool punch_obj)
 	assert_int_equal(rc, 0);
 
 	if (punch_obj)
-		rc = vos_obj_punch(args->ctx.tc_co_hdl, args->oid, ++epoch,
```

Comment:
I am wondering why it works if without the patch PR#1138 that does not change the test logic, still uses "++epoch".